index.d.ts

```diff
@@ -0,0 +1,9 @@
+type ClassValue = string | number | ClassDictionary | ClassArray;
```

Comment:
please make this match the DT typings. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a8dc31d5a2a55bf458834680d4653bc452949d84/classnames/index.d.ts